strawberryfields/decompositions.py

```diff
@@ -202,6 +212,47 @@ def clements(V, tol=1e-11):
 
     return tilist, tlist, np.diag(localV)
 
+def clements_eq5(V, tol=11):
+    r"""Clements decomposition of unitary matrix
```

Comment:
```suggestion
    r"""Performs the Clements decomposition of a Unitary complex matrix, with local
    phase shifts applied after the interferometer.
```

Reply:
Hi @ishdhand, wondering on the status of this suggested docstring change? Just curious why it was marked resolved.